styles/formative/catppuccin.user.css

```diff
@@ -81,1054 +81,665 @@
       }
     }
 
-    /* HOMEPAGE */
-    div.ClassAvatar__RootDiv-sc-1ihhtbf-0,
-    .Avatar__RootDiv-sc-r9onsm-0 > div {
-      background-color: @accent-color;
-      color: @contrast-accent-color !important;
-      span {
-        color: @contrast-accent-color;
-      }
-    }
-    div.ckhGGP {
-      background-color: @base;
-    }
+    width: 100%;
+    height: 100%;
```

Comment:
```suggestion
```

Avoid setting non-color properties if you don't have to.

Reply:
Ah, there was a white flash as the page loaded (which takes quite a while) so wanted to fix that.